qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -808,7 +808,7 @@ def test_create_s3(self):
                 'access_key': 'yyy',
                 'secret_key': 'xxx'
             })
-        self.assertStatus(200)
+        self.assertStatus(201)
```

Comment:
Hi @cah-hbaum , Is this change intended? The API test are failing for this change.

Reply:
Ah no that was not intended. I was testing some other things and that must have slipped through. Thank you for pointing it out =)

Reply:
Hi @cah-hbaum , not sure why but now the tests are expecting the status to be 201! Can you update again and try if the tests pass?

Reply:
@cah-hbaum , now I see there are two tests related to this in test_rgw.py. We need to modify the assertStatus to 201 in `test_create_swift` method as well at line no. 788. The API test are failing for this method now.

Reply:
Thanks for the suggestion, updated both again =)